source/conf.py

```diff
@@ -95,11 +95,13 @@
 # -- sphinx-JDLinker Configuration ----------------------------------------
 
 javadoc_links = {
-    "https://jd.spongepowered.org/%s/" % release: ["org.spongepowered.api"],
-    "https://configurate.aoeu.xyz/apidocs/": ["ninja.leaping.configurate"],
+    "https://jd.spongepowered.org/spongeapi/%s/" % release: ["org.spongepowered.api"],
+    "https://jd.spongepowered.org/spongeapi/0.3.0/": ["org.spongepowered.plugin"],
```

Comment:
This looks wrong to me. I already added the plugin-spi in line 101.

Reply:
Bad copy/paste and hadn't pulled in your changes at the time.